src/hotspot/share/c1/c1_LIR.cpp

```diff
@@ -351,7 +351,7 @@ LIR_OpArrayCopy::LIR_OpArrayCopy(LIR_Opr src, LIR_Opr src_pos, LIR_Opr dst, LIR_
   , _expected_type(expected_type)
   , _flags(flags) {
 #if defined(X86) || defined(AARCH64) || defined(S390) || defined(RISCV64) || defined(PPC64)
```

Comment:
Do we still need this #if?  It would be nice if we can eventually remove it, but I guess arm32 support is missing.